packages/client/hmi-client/src/components/models/tera-model-configuration.vue

```diff
@@ -178,10 +213,54 @@ const cellValueToEdit = ref({ data: {}, field: '', index: 0 });
 const selectedInitials = ref<string[]>([]);
 const selectedParameters = ref<string[]>([]);
 
+let chosenObservableIndex = 0;
+
 const configurations = computed<any[]>(
 	() => editableModelConfigs.value?.map((m) => m.configuration.semantics.ode) ?? []
 );
 
+const observables = ref<any[]>([]);
+
+function addObservable() {
+	for (let i = 0; i < editableModelConfigs.value.length; i++) {
+		if (!editableModelConfigs.value[i].configuration.semantics.ode.observables) {
+			editableModelConfigs.value[i].configuration.semantics.ode.observables = [];
+		}
+
+		editableModelConfigs.value[i].configuration.semantics.ode.observables.push({
+			id: `Obs ${editableModelConfigs.value[0].configuration.semantics.ode.observables.length + 1}`,
+			expression: 'expr'
```

Comment:
You need expressionML

Reply:
What is that - do you mean I should put a mathml expression in there?